qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -611,8 +609,8 @@ def test_du_with_path_in_args(self):
             path_prefix='')
 
         args = ['du', '/']
-        for path in path_to_files:
-            args.append(path)
+        for p in path_to_files:
+            args.append(p)
```

Comment:
why this changes is needed? even though, why not just:
```args.append(path_to_files)```

Reply:
to not overlay `path` which is already defined at the top with `from os import path`

Reply:
looks reasonable, anyways, could you please use one-liner args.append instead of for loop?